modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/diagram/NavigationsUtils.js

```diff
@@ -0,0 +1,51 @@
+export const moveRight = (container, navigationController) => {
+	const position = container.attr('transform');
+	const vai = position.match(/(-?[0-9]+[.,-\s]*)+/g);
+	const co = vai[0].split(',').map((x) => parseInt(x, 10));
+	const s = {
```

Comment:
What do `vai`, `co`, and `s` stand for?

I know that it might be "idiomatic" do use short variable names in d3, but we also have Liferay idioms that [prefer unabbreviated variable names](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/naming.md#avoid-abbreviations), so we need to strike a balance.

Reply:
i'm aware about that i edited this yet today i'll push it with the scale's fix